09_employment/compute_metrics_wage_ratio_2022.do

```diff
@@ -0,0 +1,160 @@
+/***************************
+This file imports the 2022 QCEW data and exports average weekly wage for each
+county. The file can be edited to read in data from any other year.
+
+Before using this file, download the relevant year of data from QCEW NAICS-Based Data Files, County High-Level (and select the annual summary)
+
+Programmed by Kevin Werner + updated by Kassandra Martinchek in 2023
+
+2/29/2024
+
+****************************/
+
+
+/***** update these directories *****/
+global raw "C:\Users\KMartinchek\Documents\umf-2024\mobility-from-poverty\09_employment"
```

Comment:
updated code looks fine but need to update the macros in both do files from globals to locals, otherwise the directories aren't called and code doesn't run

Reply:
Changed and re-pushed!